core/point/point_test.go

```diff
@@ -13,3 +13,114 @@ func TestCreate(t *testing.T) {
 		t.Errorf("p.Y()=%v, expected %v", gotY, 2)
 	}
 }
+
+func TestPointToSGFTranslate(t *testing.T) {
+	// First test translation from integer-point to SGF-string-point
+	testToSGFCases := []struct {
+		desc string
+		in   *Point
+		want string
```

Comment:
should have expErrSubstr or similar. See: https://github.com/otrego/clamshell/blob/master/core/sgf/parser_test.go#L23